theano/tensor/tests/test_opt.py

```diff
@@ -5393,6 +5393,32 @@ def test_assert_op_gradient():
     assert func(x_val) == 1
 
 
+class TestDivByOne(unittest.TestCase):
```

Comment:
Please update the name the name to contain IntDiv instead of Div to avoid confusion.